index.js

```diff
@@ -1329,6 +1340,48 @@ function pad(str, width) {
   return str + Array(len + 1).join(' ');
 }
 
+/**
+ * Wraps the given string with line breaks at the specified width while breaking
+ * words and indenting every but the first line on the left.
+ *
+ * @param {String} str
+ * @param {Number} width
+ * @param {Number} indent
+ * @return {String}
+ * @api private
+ */
+function wrap(str, width, indent) {
+  var regex = new RegExp('.{1,' + (width - 1) + '}([\\s\u200B]|$)|[^\\s\u200B]+?([\\s\u200B]|$)', 'g');
+  var lines = str.match(regex) || [];
+  var result = lines.map(function(line, i) {
+    if (line.slice(-1) === '\n') {
+      line = line.slice(0, line.length - 1);
+    }
+    return ((i > 0 && indent) ? Array(indent + 1).join(' ') : '') + line;
+  }).join('\n');
+  return result;
+}
+
+/**
+ * Optionally wrap the given str to a max width of width characters per line
+ * while indenting with indent spaces. Do not wrap the text when the minWidth
+ * is not hit.
+ *
+ * @param {String} str
+ * @param {Number} width
+ * @param {Number} indent
```

Comment:
The document parameter is missing `minWidth`.

Reply:
Prompted by this, I moved minWidth to an internal detail.